proto/kvrpcpb.proto

```diff
@@ -226,3 +226,14 @@ message RawDeleteResponse {
     errorpb.Error region_error = 1;
     string error = 2;
 }
+
+message RawScanRequest {
+    Context context = 1;
+    bytes start_key = 2;
+    uint32 limit = 3;
+}
+
+message RawScanResponse {
+    errorpb.Error region_error = 1;
+    repeated KvPair kvs = 2;
```

Comment:
And I don't think we need _IsTruncated_, client can decide whether to start another scan by checking if `len(kvs) == limit`.